tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Services/LoginServiceTests.cs

```diff
@@ -25,9 +26,8 @@ public async Task SignInAsync_FailedSignIn_NullContext_ReturnsExpectedResult()
             var result = await loginService.SignInAsync("user", "pass", null);
 
             Assert.NotNull(result);
-            result.IsSuccessful.Should().BeFalse();
-            result.IsTrustedReturnUrl.Should().BeFalse();
-            result.LoginHint.Should().BeNull();
+            result.IsSuccess.Should().BeFalse();
+            result.Value.Should().BeNull();
```

Comment:
The previous version of this unit test checked the other properties of the SignInResult class e.g. IsTrustedReturnUrl. Could these be added back in?

Reply:
This has now changed due to the addition of the Result class, so those values are in the result.Value (which is null)